.travis.yml

```diff
@@ -44,3 +44,13 @@ script:
   - inc/vendor/bin/phpunit
   - inc/vendor/bin/phpcs
   - composer validate --strict
+
+test:
```

Comment:
This doesn't actually seem to be getting ran by Travis as far as I can tell: https://travis-ci.org/mybb/mybb/jobs/561740143

It might be easiest to just add it as a script step:

```
- vendor/bin/phpstan analyse -l max -c inc/src/phpstan.neon src --no-interaction --no-progress
```